mysite/static/css/search/search.css

```diff
@@ -142,7 +143,7 @@ body#search #results ul { margin: 0; padding: 0; float: left; width: 99%; }
 
             body#search #results li .tags .project { padding: .3em .7em; float: left; }
 
-        body#search.actually_search #results form { float: left; width: 690px; }
+        body#search.actually_search #results form { float: left; width: 80px; }
```

Comment:
I'm a little troubled by this particular line. The commit shrinks the width of _something_ by almost 90%, but I'm not sure why this change is needed.